graylog2-web-interface/webpack.combined.config.js

```diff
@@ -16,7 +16,11 @@ const globOptions = {
   nodir: true,
 };
 
-const pluginConfigs = process.env.disable_plugins === 'true' ? [] : glob.sync(pluginConfigPattern, globOptions).map(config => `${globCwd}/${config}`);
+const pluginConfigs = process.env.disable_plugins === 'true' ? [] : glob.sync(pluginConfigPattern, globOptions).map((config) => `${globCwd}/${config}`);
+
+if (pluginConfigs.filter((config) => config.includes('graylog-plugin-cloud/server-plugin')).length > 0) {
```

Comment:
`Array.some()` will return true if there is any element which matches the condition. So no need to filter and then count the amount of elements.

```suggestion
if (pluginConfigs.some((config) => config.includes('graylog-plugin-cloud/server-plugin'))) {
```

Reply:
Thanks!